### PR TITLE
Fix issue #529 Main-Class not found error in start-client.bat

### PR DESCRIPTION
**Pull request by liuruiyiyang**

### CODE REVIEW

File: iotdb-cli/cli/bin/start-client.bat [hunk 1]
```diff
@@ -28,7 +28,7 @@ goto :eof
 REM -----------------------------------------------------------------------------
 :okClasspath
 
-"%JAVA_HOME%\bin\java" %JAVA_OPTS% -cp %CLASSPATH% %MAIN_CLASS% %*
+"%JAVA_HOME%\bin\java" %JAVA_OPTS% -cp "%CLASSPATH%" %MAIN_CLASS% %*
```
Comment (MyXOF) [inline iotdb-cli/cli/bin/start-client.bat]: Does iotdb faces same problem?if you add "", can this script run in other env?

**Comment by MyXOF** [comment]
Do other scripts face same problem?

File: iotdb-cli/cli/bin/start-client.bat (hunk 1 above)
Comment (liuruiyiyang) [reply iotdb-cli/cli/bin/start-client.bat]: iotdb server start normally

**Comment by liuruiyiyang** [comment]
I checked start-postBackClient.bat, start-server.bat, stop-postBackClient.bat, stop-server.bat for iotdb, and export-csv.bat, import-csv.bat, run-client.bat for iotdb-cli, they all executed normally.

**Pull request merged**